packages/manager/modules/cloud-connect/src/details/statistics/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,21 @@
+{
+  "cloud_connect_stats_title": "Statistiques",
+  "cloud_connect_stats_type": "Type",
+  "cloud_connect_stats_type_light": "Lumière",
+  "cloud_connect_stats_type_traffic": "Traffic",
+  "cloud_connect_stats_type_error": "Erreur",
+  "cloud_connect_stats_period": "Périodicité",
+  "cloud_connect_stats_period_hourly": "Toutes les heures",
+  "cloud_connect_stats_period_daily": "Journalier",
+  "cloud_connect_stats_period_weekly": "Hebdomadaire",
+  "cloud_connect_stats_traffic_download": "ID port {{ interfaceId }} - download",
+  "cloud_connect_stats_traffic_upload": "ID port {{ interfaceId }} - upload",
+  "cloud_connect_stats_error_download": "ID port {{ interfaceId }} - download",
+  "cloud_connect_stats_error_upload": "ID port {{ interfaceId }} - upload",
+  "cloud_connect_stats_light_in": "ID port {{ interfaceId }} - in",
+  "cloud_connect_stats_light_out": "ID port {{ interfaceId }} - out",
```

Comment:
**question:** Are there any reason to store such content into a translation file?

Reply:
Sorry I'm not sure to understand, but all text displayed into interfaces are stored into translation file ? If you change your mind is it possible to have more detail on your point of view ?